server/src/sqlz/ops/messages.ts

```diff
@@ -3,11 +3,11 @@ import Messages from '../models/messages'
 import { nanoid } from 'nanoid'
 import { QueryTypes } from 'sequelize'
 import sequelize from '../../sqlz/models'
-
+import { MessageAttributes } from '../models/messages'
 export async function getMessages(roomId, messageId) {
   if (messageId === undefined) {
     const messages = await sequelize.query(
-      'select * from "Messages" m where "roomId" = ? order by "unixTime" desc limit 25',
+      'select * from "Messages" m, "Users" u where ("roomId" = ? and m."userId" = u."userId") order by "unixTime" limit 25',
```

Comment:
- `"roomId"` should be `m."roomId"`.
- Removing the `desc` order will give you the oldest 25 messages rather than the latest, so don't remove it.
- An inner join  would be much better than cartesian product (atleast I think so).
- The `select *` selects all attributes from Messages and Users for that selected rows which also selects unnecessary columns like password, createdAt, etc while we only need the username and userId.

Try this:

```suggestion
      'select m, u.username, u."userId" from "Messages" m join "Users" u on u."userId" = m."userId" where "roomId" = ? order by "unixTime" desc limit 25',
```